testsuite/src/it/java/org/graalvm/tests/integration/utils/WhitelistLogLines.java

```diff
@@ -82,6 +82,11 @@ public Pattern[] get(boolean inContainer) {
             p.add(Pattern.compile(".*Minimum pool size: undefined/unknown.*"));
             p.add(Pattern.compile(".*Isolation level: <unknown>.*"));
             p.add(Pattern.compile(".*Maximum pool size: undefined/unknown.*"));
+            if (QUARKUS_VERSION.compareTo(new QuarkusVersion("3.27.0")) >= 0) {
+                p.add(Pattern.compile(".*JDBC fetch size: undefined/unknown.*"));
+                p.add(Pattern.compile(".*Pool: undefined/unknown.*"));
+                p.add(Pattern.compile(".*Default catalog/schema: unknown/unknown.*"));
+            }
```

Comment:
@yrodiere do you know why these are suddenly popping up? It's not happening in all cases AFAICS.